test/topics_test.rb

```diff
@@ -162,6 +162,14 @@
                      "a topic should only be an alias or a related topic, but not both"
       end
 
+      it "has existing related topics" do
+        related_topics = related_topics_for(topic)
+        related_topics.each do |related_topic|
+          assert client.topic(related_topic),
+                  "A related topic for '#{topic}' should have at least one repository tagged with it, however, '#{related_topic}' does not" 
+        end
```

Comment:
I think we can avoid the extra local variable here. Also, removing 1 character to line up correctly
```suggestion
        related_topics_for(topic).each do |related_topic|
          assert client.topic(related_topic),
                 "A related topic for '#{topic}' should have at least one repository tagged with it, however, '#{related_topic}' does not" 
        end
```